### DISCUSSION TITLE
not able to connect my react native mobile wallet with this standard dapp https://react-app.walletconnect.com/ getting "An error occured"

### DISCUSSION THREAD

**Discussion by ric344** [WalletConnect/walletconnect-monorepo#2570] [Developer Support]
### **Description**
not able to connect my react native mobile wallet with this standard dapp https://react-app.walletconnect.com/ getting "An error occured" and my wallet names showing in dashboard and giving me option to disconnect when i am refreshing. When I am doing console to connector.connected it logs undefined??

### **Technical Details**
I followed the official document like create a instance for web3wallet with ProjectId and metadata.
const connector = await Web3Wallet.init({
        core: new Core({projectId: process.env.ID}), // <- pass the shared `core` instance
        metadata: {
            name: 'Wallet',
            description: 'Wallet',
            url: 'www.anc.com',
            icons: [''],
        }
      })
console.log("connector:", connector);
await connector.core.pairing.pair({ uri: uri });

console.log("!connector.connected:", connector.connected);// logs to undefined.

After that creating session proposal and approving:
connector.on("session_proposal", async (proposal) => {
        const namespaces= {};
        const { id, params } = proposal;
        const {requiredNamespaces, relays} = params;
        Object.keys(requiredNamespaces).forEach(key => {
          const accounts = [];
          requiredNamespaces[key].chains.map(chain => {
            [wallet].map(acc => accounts.push(`${chain}:${acc}`));
          });
  
          namespaces[key] = {
            accounts,
            methods: requiredNamespaces[key].methods,
            events: requiredNamespaces[key].events,
          };
        });
        const session = await connector.approveSession({
          id: proposal.id,
          namespaces: namespaces,
      });
Also I am able to console the getPairings and it is showing correctly.
Can you suggest me what I am doing wrong? Thanks

**Comment by ignaciosantise** [comment]
Hey @ric344 👋 A few questions related to this:
- Can you check if you can get a better error?
- Does the error show in https://react-app.walletconnect.com/  or in your app?
- `connector.connected` will always log `undefined` because `.connected` prop doesn't exist
- You can also check [here](https://github.com/WalletConnect/react-native-examples/blob/595e1cf1ee29c8ebbd889327943dca312749a8cc/wallets/rn_cli_wallet/src/screens/HomeScreen.tsx#L125) how to handle a `session_proposal`

Finally, can you provide a repo with an example that reproduces the bug? you can fork [this example](https://github.com/WalletConnect/react-native-examples/tree/main/wallets/rn_cli_wallet) and add your code there
